Algorithm/QCAlgorithm.Plotting.cs

```diff
@@ -52,7 +52,7 @@ public void AddChart(Chart chart)
         {
             if (!_charts.ContainsKey(chart.Name))
             {
-                _charts.Add(chart.Name, chart);
+                _charts.TryAdd(chart.Name, chart);
```

Comment:
Should we remove the `ContainsKey` check?

Reply:
Sure, it's no longer needed also we never check the bool return value, so we can safely remove the check.